config/locales/client.en.yml

```diff
@@ -3472,6 +3474,14 @@ en:
         title: "This topic is a personal message"
         help: "This topic is a personal message"
     posts: "Posts"
+    pending_posts:
+      label:
+        zero: "Pending"
+        one: "Pending (%{count})"
+        other: "Pending (%{count})"
+      empty:
+        title: "You don’t have any posts awaiting moderation"
```

Comment:
Not sure what the text should be when there is no more pending posts to display.